pages/use-solid.md

```diff
@@ -35,15 +35,22 @@ The following table lists some Pod Provider:
 
 | Provider                                           | Responsible for Domain Name and Terms                                      | Responsible for Hosting                       | Hosting Location | Solid Implementation                                        |
 |----------------------------------------------------|----------------------------------------------------------------------------|-----------------------------------------------|------------------|-------------------------------------------------------------|
-| [Inrupt Pod Spaces](https://start.inrupt.com/) | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | Germany              | [ESS](https://inrupt.com/products/enterprise-solid-server/) |
+| [Inrupt Pod Spaces](https://start.inrupt.com/) | [Inrupt, Inc.](https://www.inrupt.com/terms-conditions)                        | [Amazon](https://aws.amazon.com)              | Germany              | [ESS](https://inrupt.com/products/enterprise-solid-server/) |
```

Comment:
As of writing this https://web.archive.org/web/20240517095415/https://start.inrupt.com/profile :

>PodSpaces allows developers to get a WebID and a Solid Pod in order to create apps using Inrupt's Enterprise Solid Server

That clearly seems to be a provider/service intended to function as a proprietary system, effectively creating a vendor lock-in scenario. This is not aligned with the goals of the Solid Project, as exemplified by the statement above the Pod Providers table:

>With Solid, you won't be tied to the provider you choose now. That is, you have the ability to move your data elsewhere if you want.

In other words, the goal is to promote apps that can work with any Solid server conforming to the specifications developed through open standards bodies.

So, consider:

```suggestion
```

Reply:
> With Solid, you won't be tied to the provider you choose now. That is, you have the ability to move your data elsewhere if you want.

This is not true, and has never been true.

> This is not aligned with the goals of the Solid Project

Which goals?

> That clearly seems to be a provider/service intended to function as a proprietary system

Shouldnt solid work with both open and proprietary system?

Reply:
I have resolved everything except this one, until we have discussed / decided

Reply:
>This is not true, and has never been true.

If you believe this to be not true, please follow-up with a correction by making a new PR.

Moreoever, please also note that the quoted text was introduced and revised, in particular at:

* https://github.com/solid/solidproject.org/commit/c625071532507352bf4789c98fb980f3a23254a1
* https://github.com/solid/solidproject.org/commit/50fcdfccf23f84fd9e071035090f78764f3483b4

in fact by members of Inrupt. If they've made an error, please again, follow with a correction.

In this context, I was merely referring to the existing text on the website.

---

As Matthias has already considered and integrated some my other change requests, I am now dismissing my review.

Reply:
OK @csarven if you think this is all good now, can you give an 'approve' review and merge?